onnx2tf/onnx2tf.py

```diff
@@ -666,6 +667,7 @@ def convert(
         converter = tf.lite.TFLiteConverter.from_concrete_functions(
             [concrete_func]
         )
+        converter.optimizations = [tf.lite.Optimize.DEFAULT]
```

Comment:
Passing `[tf.lite.Optimize.DEFAULT]` to the optimizer results in INT8 quantization. I reject this modification because I clearly separate the meaning of INT8 quantization from that of generating a Float32 model. Explicitly use the `-oiqt, --output_integer_quantized_tflite` option to perform INT8 quantization.
![image](https://user-images.githubusercontent.com/33194443/207108811-d3bb6f26-4d1c-4b54-a4fd-eb2724df9f93.png)

Reply:
thanks. will revert

Reply:
reverted.